docs/source/conf.py

```diff
@@ -20,8 +20,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'pyUmbral'
-copyright = '2018, Michael Egorov, Justin Myles Holmes, David Nuñez, John Pacific, Kieran Prasch'
-author = 'Michael Egorov, Justin Myles Holmes, David Nuñez, John Pacific, Kieran Prasch'
+copyright = u'2018, Michael Egorov, Justin Myles Holmes, David Nuñez, John Pacific, Kieran Prasch'
```

Comment:
Actually, shouldn't the copyright be '2019, NuCypher'?

Reply:
Yeah, and we still have the issue of having it on all file headers. Let me add an additional commit to resolve all that.

Reply:
✔️